Jellyfin/Core/FullScreenManager.cs

```diff
@@ -0,0 +1,197 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+using Jellyfin.Utils;
+using Windows.Data.Json;
+using Windows.Graphics.Display.Core;
+using Windows.UI.ViewManagement;
+
+namespace Jellyfin.Core
```

Comment:
file scoped namespaces are required now. If you merge from master it should also show this as an error now